### PR TITLE
Put newlines between added annotations

### PR DESCRIPTION
**Pull request by matozoid**
See #1319

### CODE REVIEW

File: javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ private static CsmElement mix(CsmElement... elements) {
     }
 
     private static CsmElement memberAnnotations() {
-        return list(ObservableProperty.ANNOTATIONS, none(), none(), newline());
+        return list(ObservableProperty.ANNOTATIONS, newline(), none(), newline());
```
Comment (ftomassetti) [inline javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java:68]: Yes, it seems correct (I would love named parameters here...)

File: javaparser-core/src/main/java/com/github/javaparser/printer/concretesyntaxmodel/CsmMix.java [hunk 2]
```diff
@@ -10,7 +10,8 @@
 /**
  * A group of elements that could be in any order.
  */
-public class CsmMix implements CsmElement {private List<CsmElement> elements;
+public class CsmMix implements CsmElement {
```
Comment (ftomassetti) [inline javaparser-core/src/main/java/com/github/javaparser/printer/concretesyntaxmodel/CsmMix.java:13]: Thanks

File: javaparser-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/transformations/ast/body/AnnotationMemberDeclarationTransformationsTest.java [hunk 3]
```diff
@@ -50,7 +49,7 @@ protected AnnotationMemberDeclaration consider(String code) {
     // Name
 
     @Test
-    public void changingName() throws IOException {
+    public void changingName() {
```
Comment (ftomassetti) [inline javaparser-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/transformations/ast/body/AnnotationMemberDeclarationTransformationsTest.java:52]: Thanks for removing the useless throws declarations

**Review by ftomassetti — APPROVED** [review]
All good!

File: javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java (hunk 1 above)
Comment (matozoid) [reply javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java:68]: We can convert everything to Kotlin.

File: javaparser-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/transformations/ast/body/AnnotationMemberDeclarationTransformationsTest.java (hunk 3 above)
Comment (matozoid) [reply javaparser-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/transformations/ast/body/AnnotationMemberDeclarationTransformationsTest.java:52]: Oh, I just hit "clean file" or something in Intellij ;-)

File: javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java (hunk 1 above)
Comment (ftomassetti) [reply javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java:68]: That would be brilliant!

Comment (matozoid) [reply javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java:68]: Just kidding :-D

**Pull request merged**